packages/playwright-core/src/protocol/protocol.yml

```diff
@@ -3032,4 +3031,5 @@ JsonPipe:
         message: json
 
     closed:
+      parameters:
```

Comment:
this seemed to be like a bug in the YAML where the errors didn't get correctly plumped through.